common/src/main/java/com/wynntils/utils/LongDateFormatter.java

```diff
@@ -4,53 +4,45 @@
  */
 package com.wynntils.utils;
 
-import com.google.common.collect.ImmutableMap;
-import java.text.DateFormat;
 import java.text.FieldPosition;
-import java.text.ParsePosition;
-import java.util.Calendar;
-import java.util.Date;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.Period;
+import java.time.ZoneOffset;
+import java.time.temporal.ChronoUnit;
+import java.util.LinkedHashMap;
 import java.util.Map;
-import java.util.TimeZone;
-
-public class LongDateFormatter extends DateFormat {
-    private static final Map<Integer, String> MAPPINGS = ImmutableMap.of(
-            Calendar.YEAR, "y",
-            Calendar.MONTH, "mo",
-            Calendar.DAY_OF_MONTH, "d",
-            Calendar.HOUR_OF_DAY, "h",
-            Calendar.MINUTE, "m",
-            Calendar.SECOND, "s");
-
-    public LongDateFormatter() {
-        setCalendar(Calendar.getInstance());
-        setTimeZone(TimeZone.getTimeZone("UTC"));
-    }
 
+public class LongDateFormatter extends DateFormatter {
```

Comment:
Maybe we can merge this with SimpleDateFormatter? EIther by adding a "long" boolean in the constructor, or by adding a parameter in the method.

Reply:
Sure, we'll just have the one DateFormatter